examples/create2/test/create2.test.js

```diff
@@ -12,20 +12,4 @@ describe("Create2", function () {
   it("should return an instantiated factory", async function () {
     assert.isDefined(deployResults.create2);
   });
-
-  it("should have deployed the foo contract", async () => {
-    const address = deployResults.fooEvent.deployed;
```

Comment:
Am I following right?

We need to remove these tests because we no longer return events, which would allow us to get the deployed contracts address. We intend to implement a way of converting a returned event param to a ContractFuture within a module, but that doesn't exist yet.

Reply:
No, it does exist and that's what we are returning instead, but the rest is correct